packages/doenetml-worker-rust/lib-doenetml-core/src/core/props/data_query/data_query.rs

```diff
@@ -170,4 +209,6 @@ pub struct ComponentTypeDataQueryFilter {
 pub enum DataQueryFilterComparison {
     Equal,
     NotEqual,
+    // TODO: this is not the right way to do this. Cleanup how to build queries.
```

Comment:
With  `ContentFilter::HasPropMatchingProfileAndCondition`, is `ProfilePresent` needed anymore? If we merge the two types of filtering, maybe none of the `DataQueryFilterComparison` will be needed anymore.

Reply:
The comparions should not be used anywhere anymore. I will delete them.